psutil/_common.py

```diff
@@ -32,10 +32,14 @@
 else:
     enum = None
 
+
+PY3 = sys.version_info[0] == 3
```

Comment:
I suggest `>=` else sometime in the future Python 4 is released and no one knows why psutil is broken in cygwin.

Reply:
Agree; thanks for pointing that out.  In fact instead of a `PY3` variable it'd be better to use `PY2` for Python < 3.